libr/core/cmd_type.c

```diff
@@ -24,6 +24,7 @@ static const char *help_msg_t[] = {
 	"tn", "[?] [-][addr]", "manage noreturn function attributes and marks",
 	"to", " -", "Open cfg.editor to load types",
 	"to", " <path>", "Load types from C header file",
+	"toe", "", "Open cfg.editor to edit types",
```

Comment:
isnt toe taking an argument to specify the name of the struct to edit?.. well, struct or enum or function, or whatever its named like this

Reply:
Not yet , it's still not implemented :D , will do it today !